run.sh

```diff
@@ -40,9 +40,27 @@ export RUST_BACKTRACE=1
 dataDir=$PWD/target/"$(basename "$0" .sh)"
 
 set -x
-solana-keygen -o "$dataDir"/config/leader-keypair.json
-solana-keygen -o "$dataDir"/config/leader-vote-account-keypair.json
-solana-keygen -o "$dataDir"/config/leader-stake-account-keypair.json
+leader_keypair="$dataDir/config/leader-keypair.json"
+if [ -e "$leader_keypair" ]
```

Comment:
Oh instead of 
```sh
if [ -e "$leader_keypair" ]
then
```
please use this form:
```sh
if [[ -e "$leader_keypair" ]]; then
```
because
1. less lines of code
2. `[[` is a bash built-in

Reply:
also, in the built-in, the quotes around `$leader_keypair` are unnecessary